src/util.cc

```diff
@@ -1091,8 +1091,12 @@ static string read_exe_dir() {
 }
 
 string exe_directory() {
-  static string exe_path = read_exe_dir();
-  return exe_path;
+  string resource_path = Flags::get().resource_path;
+  if (resource_path.empty()) {
+    static string exe_path = read_exe_dir() + "../";
+    return exe_path;
+  }
+  return resource_path;
```

Comment:
This is basically fine but `exe_directory()` no longer means "the directory containing the rr exe". Now it means the path relative to which resources are found. So I think it should actually be renamed to `resource_path()`.

Reply:
Makes sense.  One moment....

Reply:
And done.  Thanks!